include/deal2lkit/parsed_grid_generator.h

```diff
@@ -77,6 +77,45 @@ class ParsedGridGenerator : public ParameterAcceptor
    */
   Triangulation<dim, spacedim> *serial();
 
+  /**
+   * Generate the grid. It is implemented is such a way there are no error due
```

Comment:
Remove the phrase "It is implemented is such a way there are no error due to the particual choice of dim and spacedim (some grid requires dim==spacedim and others dim+1=spacedim).

This should be the default... ;)